static/app/components/events/contexts/unity/index.tsx

```diff
@@ -0,0 +1,41 @@
+import {Fragment} from 'react';
+
+import ContextBlock from 'sentry/components/events/contexts/contextBlock';
+import {Event, UnityContext} from 'sentry/types';
+
+import {geKnownData, getUnknownData} from '../utils';
+
+import {getUnityKnownDataDetails, unityKnownDataValues} from './getUnityKnownDataDetails';
+
+type Props = {
+  data: UnityContext | null;
+  event: Event;
+};
+
+export function UnityEventContext({data, event}: Props) {
```

Comment:
This file looks to be 99% the same as `memoryInfo` and some others, any chance we could have one reusable one? At least for similar contexts?